www/statusbar.js

```diff
@@ -153,6 +156,21 @@ var addStatusBarDataElement = function(){
 }
 
 
+var injectViewportMetaTag = function(){
+
+    if(device.platform.toLowerCase().indexOf(IOS_PLATFORM) == 0 && device.version.split(".")[0].indexOf(IOS_11_VERSION) == 0){
```

Comment:
It's better to validate for >= 11 instead of only for iOS 11, otherwise, with iOS 12 (when released) it should break